pallets/astar-xcm-benchmarks/src/fungible/mod.rs

```diff
@@ -0,0 +1,22 @@
+// This file is part of Astar.
+
+// Copyright (C) 2019-2023 Stake Technologies Pte.Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// Astar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Astar is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Astar. If not, see <http://www.gnu.org/licenses/>.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+
+pub mod assets;
+pub mod balances;
```

Comment:
Are two completely separate modules really needed?

Seems like lots of copy/paste, for mock.rs and mod.rs

Reply:
I wanted our pallet to have consistent style with the original pallet so I used the same format. We can also merge them into one but that would mean changing the variable names and name of benchmarks and some logic which will be different from the original pallet. 
Let me know what you think.